.changelog/2098.feature.md

```diff
@@ -0,0 +1,6 @@
+node: Add automatic TLS certificate rotation support
```

Comment:
Probably open an issue in https://github.com/oasislabs/docs once this is merged, so we wont forget adding/updating docs

Reply:
Will do.